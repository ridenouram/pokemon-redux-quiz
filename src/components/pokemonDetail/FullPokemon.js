import React from 'react';
import PropTypes from 'prop-types';

function FullPokemon({ pokemon }) {
  return (
    <section>
      <h1>{pokemon.pokemon}</h1>
      <img src={pokemon.url_image} />
      <div>
        <h3>Attack: {pokemon.attack}</h3>
        <h3>Defense: {pokemon.defense}</h3>
        <h3>HP: {pokemon.hp}</h3>
      </div>
      <a href={pokemon.pokedex}>More Information</a>
    </section>
  );
}

FullPokemon.propTypes = {
  pokemon: PropTypes.object.isRequired
};

export default FullPokemon;
