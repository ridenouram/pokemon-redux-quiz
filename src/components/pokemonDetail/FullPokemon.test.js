import React from 'react';
import { shallow } from 'enzyme';
import FullPokemon from './FullPokemon';

describe('FullPokemon component', () => {
  it('snapshot', () => {
    const wrapper = shallow(<FullPokemon pokemon={{}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
