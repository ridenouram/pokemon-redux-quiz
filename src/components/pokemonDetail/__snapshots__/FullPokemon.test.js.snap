// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullPokemon component snapshot 1`] = `
<section>
  <h1 />
  <img />
  <div>
    <h3>
      Attack: 
    </h3>
    <h3>
      Defense: 
    </h3>
    <h3>
      HP: 
    </h3>
  </div>
  <a>
    More Information
  </a>
</section>
`;
